blog/2019-04-25-pulsar-debezium-tutorial.adoc

```diff
@@ -0,0 +1,242 @@
+= Tutorial for Pulsar Debezium Connecttor
+jiazhai, StreamNative
+:awestruct-tags: [ discussion, examples  ]
+:awestruct-layout: blog-post
+
+https://debezium.io[Debezium] is an open source project for change data capture (CDC). It is built on Kafka Connect and supports multiple databases, such as MySQL, MongoDB, PostgreSQL, Oracle, and SQL Server. 
+http://pulsar.apache.org[Apache Pulsar] provides a http://pulsar.apache.org/docs/en/2.3.0/io-cdc-debezium[source connector] over Kafka connect, so you can leverage Debezium to stream changes from your databases. 
+This tutorial walks you through using Pulsar source connector to store MySQL data changes. 
+
+## Tutorial Steps
+This tutorial is similar to https://debezium.io/docs/tutorial[debezium tutorial], except that storage of event streams is changed from Kafka to Pulsar.
+It mainly includes six steps: 
+1. Start a MySQL server;
+2. Start standalone Pulsar service;
+3. Start Pulsar connector. Pulsar connector reads database changes existing in MySQL server;
```

Comment:
So I'm stumbling a bit upon "Pulsar connector". Technically, it's the Debezium MySQL connector. Wondering how this could be expressed so it's correct while not be too verbose. Perhaps "Start the Debezium connector in Pulsar IO"?